torchtune/modules/low_precision/nf4_linear.py

```diff
@@ -18,27 +18,29 @@ class FrozenNF4Linear(nn.Linear):
     NF4Tensor as its weight. This class also freezes its ``weight`` parameter
     and is meant to be used as the base Linear layer for modeling
     use cases such as QLoRA where base model parameters are frozen.
-    NOTE: biases are currently not supported.
 
     Args:
         in_dim (int): input dimension
         out_dim (int): output dimension
         device (Optional[torch.device]): device to use for the underlying weight. If ``None``, uses the default
             device given by `torch.get_default_device()`.
+        bias (bool): whether to include bias in the original linear layer. Default: False
         **kwargs: any additional arguments to pass to the underlying Linear layer.
 
-    Raises:
-        RuntimeError: if ``bias`` is set to ``True``
     """
 
     def __init__(
-        self, in_dim: int, out_dim: int, device: Optional[torch.device] = None, **kwargs
+        self,
+        in_dim: int,
+        out_dim: int,
+        device: Optional[torch.device] = None,
+        bias: bool = False,
+        **kwargs,
     ):
-        if "bias" in kwargs and kwargs.pop("bias"):
-            raise RuntimeError("FrozenNF4Linear does not currently support biases!")
-
-        super().__init__(in_dim, out_dim, device=device, bias=False, **kwargs)
+        super().__init__(in_dim, out_dim, device=device, bias=bias, **kwargs)
         self.weight.requires_grad_(False)
+        if self.bias is not None:
+            self.bias.requires_grad_(False)
         self.nf4_weight = to_nf4(self.weight)
```

Comment:
nit: do we want this "self." here?

Reply:
Yeah good point, maybe not strictly necessary